experimenter/experimenter/nimbus-ui/src/components/PageResults/TableWeekly/index.test.tsx

```diff
@@ -76,4 +76,43 @@ describe("TableWeekly", () => {
 
     expect(screen.getAllByText(ERROR_TEXT)).toHaveLength(2);
   });
+
+  it("sorts week indices numerically", () => {
+    const modifications = {
+      treatment: {
+        is_control: false,
+        branch_data: {
+          other_metrics: {
+            retained: WEEKLY_EXTRA_LONG,
+          },
+        },
```

Comment:
I'm sorry but I led you astray with my last suggestion! It looks like you actually need another one of the modifications you had here -- you'll want to add back the `identity` metric and set it to `WEEKLY_EXTRA_LONG` as well because `identity` is used in one of the sub-components of `TableWeekly`.